### PR TITLE
#4962 - Bond tool and Erase tool buttons have to be disabled in Sequeince mode (since they are not applicable)

### PR DESCRIPTION
**Pull request by Guch1g0v**
## How the feature works? / How did you fix the issue?
(Screenshots, videos, or GIFs, if applicable)

## Check list
- [ ] unit-tests written
- [ ] e2e-tests written
- [ ] documentation updated
- [x] PR name follows the pattern `#1234 – issue name`
- [x] branch name doesn't contain '#'
- [x] PR is linked with the issue
- [x] base branch (master or release/xx) is correct
- [x] task status changed to "Code review"
- [x] reviewers are notified about the pull request

### CODE REVIEW

File: packages/ketcher-macromolecules/src/components/LeftMenuComponent/LeftMenuComponent.tsx [hunk 1]
```diff
@@ -25,6 +25,7 @@ const shortcuts =
 export function LeftMenuComponent() {
   const activeTool = useAppSelector(selectEditorActiveTool);
   const editor = useAppSelector(selectEditor);
+  const isSequenceMode = useLayoutMode() === 'sequence-layout-mode';
```
Comment (rrodionov91) [inline packages/ketcher-macromolecules/src/components/LeftMenuComponent/LeftMenuComponent.tsx:28]: Please use constant ModeTypes here.

**Review by rrodionov91 — CHANGES_REQUESTED** [review]

**Review by rrodionov91 — APPROVED** [review]

**Review by rrodionov91 — CHANGES_REQUESTED** [review]
Please fix failed tests

**Review by AlexeyGirin — APPROVED** [review]
Approved

File: ketcher-autotests/tests/Macromolecule-editor/Sequence-Mode/sequence-mode-selection.spec.ts [hunk 2]
```diff
@@ -204,7 +203,7 @@ test.describe('Sequence mode selection for view mode', () => {
     await takeEditorScreenshot(page);
   });
 
-  test('Select a nucleotide or a group of nucleotides, then switch to another tool (excluding Erase). Confirm that selection is cleared.', async ({
+  test('Select a nucleotide or a group of nucleotides, then click in the middle of the screen. Confirm that selection is cleared.', async ({
```
Comment (rrodionov91) [inline ketcher-autotests/tests/Macromolecule-editor/Sequence-Mode/sequence-mode-selection.spec.ts:206]: The name of test was changed, so it now generates screenshots with new name. Please delete old screenshots then.

**Review by rrodionov91 — CHANGES_REQUESTED** [review]

**Review by rrodionov91 — APPROVED** [review]

**Pull request merged**